client/src/App.js

```diff
@@ -1,13 +1,24 @@
-
+import React from "react"
 import './App.css';
 import Routing from "./routes/index";
 
-
 function App() {
+
+  const [data, setData] = React.useState(null);
```

Comment:
shouldn't be React.useState just useState

Reply:
removed React.useState and React.useEffect